ui/components/app/snaps/snap-install-warning/snap-install-warning.js

```diff
@@ -126,6 +178,34 @@ export default function SnapInstallWarning({
       warning.permissionName === 'snap_getBip44Entropy',
   );
 
+  const warningPermissionCells = {
```

Comment:
This seems like it is complicating things a bit more than they need to be. Hmm, we should maybe refactor a bit 🤔

Reply:
Refactored. Works now without this.